components/infobox/commons/infobox_league.lua

```diff
@@ -454,4 +455,25 @@ function League:_getPageNameFromChronology(item)
 	return mw.text.split(item, '|')[1]
 end
 
+
+-- Given a series query its abbreviation if abreviation is not set manually
+function League:_fetchAbbreviation()
+	if not String.isEmpty(self.args.abbreviation) then
+		return self.args.abbreviation
+	elseif String.isEmpty(self.args.series) then
+		return nil
+	end
+
+	local series = string.gsub(mw.ext.TeamLiquidIntegration.resolve_redirect(self.args.series), ' ', '_')
+	local seriesData = mw.ext.LiquipediaDB.lpdb('series', {
+			conditions = '[[pagename::' .. string.gsub(mw.ext.TeamLiquidIntegration.resolve_redirect(series), ' ', '_') .. ']]'
+				.. ' AND [[abbreviation::>]]',
+			query = 'abbreviation',
+			limit = 1
+		})
+	if seriesData and seriesData[1] then
```

Comment:
```suggestion
	if type(seriesData) == 'table' and seriesData[1] then
```
as I'm pretty sure that LPDB never returns nil, but it can return a string if the `conditions` is invalid

Reply:
true